Kostiantyn_Nifantiev/01/ArrayDifference.py

```diff
@@ -0,0 +1,36 @@
+def array_dif_v1(first_array, second_array):
+    """
+        This version accumulates result by checking every number
+        in first array for presence in second one
+    """
+    result = list()
+
+    for element in first_array:
+
+        if not second_array.count(element): result += element
+
+    return result
+
+def array_dif_v2(first_array, second_array):
+    """
+        This version removes elements from copy of initial array
+    """
+    result = set(first_array)
+
+    for element in second_array: 
+        try: result.remove(element)
```

Comment:
Avoid using exceptions handling for now. You should do a new line after colon ":".